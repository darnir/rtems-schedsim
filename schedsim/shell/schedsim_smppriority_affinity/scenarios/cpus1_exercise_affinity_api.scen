#
#  Verify that affinity on 1 core
#
rtems_init 1

# GOAL: 0x0a010001 with affinity
echo "=== Create and start 0x0a010001 ==="
task_create TA1 3 -a 1

# GOAL: 0x0a010001 is now heir and executing on CPU 0
cpus TA1
