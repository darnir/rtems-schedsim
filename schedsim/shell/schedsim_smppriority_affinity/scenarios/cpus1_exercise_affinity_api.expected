Script File               : scenarios/cpus1_exercise_affinity_api.scen
verbose                   : 1
==> 1: #
==> 2: #  Verify that affinity on 1 core
==> 3: #
==> 4: rtems_init 1
  CPU 0: Thread Heir 0x09010001 priority=255
  CPU 0: Thread Executing 0x09010001 priority=255
==> 5: 
==> 6: # GOAL: 0x0a010001 with affinity
==> 7: echo "=== Create and start 0x0a010001 ==="
=== Create and start 0x0a010001 ===
==> 8: task_create TA1 3 -a 1
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010001, priority=3
Task (0x0a010001) Set affinity=0x00000001
Task (TA1) starting: id=0x0a010001, priority=3
  CPU 0: Thread Heir 0x0a010001 priority=3
  CPU 0: Thread Executing 0x0a010001 priority=3
==> 9: 
==> 10: # GOAL: 0x0a010001 is now heir and executing on CPU 0
==> 11: cpus TA1
=== CPU Status
          EXECUTING      /   HEIR             / SWITCH NEEDED
  CPU 0: 0x0a010001 @  3 / 0x0a010001 @  3          false
=== End of Ready Set of Threads
*** ERROR in scenario -- unknown task TA1
